src/Compilers/CSharp/Portable/LanguageVersion.cs

```diff
@@ -212,6 +229,7 @@ public static LanguageVersion MapSpecifiedToEffectiveVersion(this LanguageVersio
             switch (version)
             {
                 case LanguageVersion.Latest:
+                    return LanguageVersion.CSharp7_1;
```

Comment:
I think this file is very easy to edit. After the recent rectoring (by you), all our version work is done here, and adding a new version is simply editing 3~4 lines in that file, plus the required testing.

---
In reply to: [106483298](https://github.com/dotnet/roslyn/pull/17894#discussion_r106483298) [](ancestors = 106483298)